hydra-node/src/Hydra/Ledger/Cardano.hs

```diff
@@ -429,6 +434,14 @@ genAddressInEra networkId =
 genValue :: Gen Value
 genValue = fmap ((lovelaceToValue $ Coin 10_000_000) <>) (scale (`div` 10) $ fromLedgerValue <$> arbitrary)
 
+genSomeTokens :: Gen Value
+genSomeTokens = do
+  tokenName <- arbitrary
+  policyId <- arbitrary
+  n <- getPositive <$> arbitrary
+  quantity <- choose (1, n)
+  pure $ valueFromList [(AssetId policyId tokenName, Quantity quantity)]
```

Comment:
Should: document why this is different than `genValue` (and also name this with `genValue...`)

We have so many generators already and while specific ones are needed sometimes, I don't see why we need this one? Maybe because quantities are too large on `genValue`? If yes, why don't we use `sized` for a size-bound value?